### ISSUE TITLE
Request "Sign and Post" views

### ISSUE THREAD

**Issue by kmw101** [iov-one/ponferrada#31]
Handling requests transactions will imply some steps:
- [x] The main view will have two buttons, one for rejecting named "Reject", another one for accepting called "accept".
~- [ ]  If the user clicks on accept, is redirected to account status view, and a **pending** txs is shown until it is mined, then tx will be shown in txs box.~
- [x]  If the user clicks on reject, a second view is going to be shown, as a confirmation step, where a checkbox is shown for blocking sender to send more requests to the extension.
- [x]  If the user finally rejects the tx, it should be shown in the tx box in the account status view, after confirming rejection (remember as @albertandrejev suggested the button should be named "reject") the extension is redirected to account status view.

NOTE:
- If the user clicks the checkbox the sender should be added to the blocked lists, to be implemented in #29 after #156 
- **Please read**: The ux flow described: https://projects.invisionapp.com/share/PRPRZCQ9MHY#/screens/338674184 is just an example, as discussed in the meeting we should remove some stuff if you have doubts or you have forgotten something just ask in slack.

Task decomposition:
- [x] Storybook and main view #170;
- [x] Storybook and reject view #170; 
- [x] Add route and DOM tests #185;

**Comment by apanizo** [comment]
@albertandrejev @abefernan forget the "recurrent" word in the invision's mock. We won't do recurrent txs at all.

**Comment by webmaster128** [comment]
Closing since this is done
